azurelinuxagent/common/AgentGlobals.py

```diff
@@ -1,4 +1,4 @@
-# Microsoft Azure Linux Agent
+# Microsoft Azure Linux Agent # pylint: disable=C0103
```

Comment:
should this one be here? it's within a comment

Reply:
Ah. This in particular might be problematic.
I think the exact error this comment is suppressing is that the module `AgentGlobals` does not follow correct naming conventions.

Though this does in fact disable that error (showing that pylint happily accepts this command within a comment), any additional `invalid-name<C0103>` errors later within this file would also be disabled, which is less than ideal...

I do not see a work around here though.

Note: `<code> # pylint: disable=<some_err> # pylint: disable=<some_other_err>` actually only disables `<some_other_err>` (and not also `<some_err>`).  This is obviously something we have to be weary of, but it makes me think that pylint is simply looking at the end of each line for this pattern.

Reply:
OK, we can leave it as is for now. This will be one easy to fix.